src/pages/Home/Filters/Filters.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { Select } from '../../../components'
+import { movieFilters, sortMovies } from '../../../components/constants'
+import cn from 'classnames'
+import styles from './Filters.css'
+
+const Filters = () => {
+  const [checkedTab, setCheckedTab] = useState('all');
```

Comment:
it will be better
```js
const [checkedTab, setCheckedTab] = useState(movieFilters[0]);
```

Reply:
https://github.com/Azaliyaa/netflix-roulette/pull/1/commits/d5e77e4870f4012c2090494a023691ab138bc616